lib/kb_maxbin/kb_maxbinImpl.py

```diff
@@ -22,7 +22,7 @@ class kb_maxbin:
     # state. A method could easily clobber the state set by another while
     # the latter method is running.
     ######################################### noqa
-    VERSION = "1.0.0"
+    VERSION = "1.0.1"
```

Comment:
please run make. it should be updated to 1.1.0

Reply:
Actually, given that there's a version change of the wrapped tool, I agree that 1.1.0 is better than 1.0.1 as the updated module verison